includes/admin.php

```diff
@@ -167,6 +175,10 @@ function output_meta_box( WP_Post $post ) {
 		</select>
 		<em><?php esc_html_e( "If you don't know what this is, leave it as is.", 'hm-redirects' ); ?></em>
 	</p>
+	<p>
+		<input type="checkbox" name="hm_redirects_parameters" id="hm_redirects_parameters" value="1" <?php echo ( $preserve_url ? 'checked' : ''  );?> />
+		<label for="hm_redirects_parameters"><?php esc_html_e( 'Preserve URL parameters?', 'hm-redirects' ); ?></label>
```

Comment:
I think this needs a description as well to explain what exactly it does; something like
> By default, query parameters (e.g. `?utm_campaign=x`) will be removed when redirecting. Enable this option to preserve these parameters when redirecting.

Reply:
Thank you for this @rmccue, I have gone and added the description